tests/integration/tier0/check-user-response/test_user_response.py

```diff
@@ -0,0 +1,50 @@
+from envparse import env
+
+
+def test_check_user_response_user_and_password(convert2rhel):
+
+    # Run c2r registration with no username and password provided
+    # check for user prompt enforcing input, then continue with registration
+    with convert2rhel(
+        "-y --no-rpm-va --serverurl {}".format(
+            env.str("RHSM_SERVER_URL"),
+        )
+    ) as c2r:
+        c2r.expect_exact(" ... activation key not found, username and password required")
+        c2r.expect_exact("Username")
+        c2r.sendline()
+        assert c2r.expect_exact(["Username", "Password"]) == 0
```

Comment:
Just a question, are the asserts here necessary?

Reply:
I believe so, as I would like to check if the assertion wouldn't raise `assert 1 == 0` which is the unwanted behavior.

Reply:
As far as I know the `expect()` returns the index of the first match it gets. So in this case you are checking that the "Username" will be displayed and not the "Password" right?

Reply:
If that's the case I would add a comment above this line, explaining that we want to check the presence of the "Username" and raise am error if the "Password" would appear instead.

Reply:
Exactly. Alright will do that.